CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+# 0.0.114 - XXX - "Faster Async BOLT12 Retries"
+
+## API Updates
+ * `InvoicePayer` has been removed and its features moved directly into
+   `ChannelManager`. As such it now requires a simplified `Router` and supports
+   `send_payment_with_retry` (and friends) (#1812, #1916, #1929, #2007, etc).
```

Comment:
this made me think `pay_invoice` would be in `ChannelManager`, would be nice to clarify that those are still in the crate